training/sample_training_data/README.md

```diff
@@ -0,0 +1,3 @@
+## Sample training data
+
+If this directory contains any files (other than this README) you can test-drive the model training process.
```

Comment:
I think we can add a little more detail about what the sample data is (how it was created, annotated etc) - with link to data prep step / Cloud Annotations tool where relevant?